src/mame/drivers/seattle.cpp

```diff
@@ -1400,7 +1400,10 @@ static INPUT_PORTS_START( seattle_analog )
 
 	PORT_MODIFY("SYSTEM")
 	PORT_BIT( 0x0004, IP_ACTIVE_LOW, IPT_START1 ) PORT_NAME("Start Button")
-	PORT_BIT( 0x0020, IP_ACTIVE_LOW, IPT_UNUSED )
+	PORT_BIT( 0x0620, IP_ACTIVE_LOW, IPT_UNUSED )
+
+	PORT_MODIFY("IN2")
+	PORT_BIT( 0xffff, IP_ACTIVE_LOW, IPT_UNUSED  )
```

Comment:
Are these really `IPT_UNUSED` or just `IPT_UNKNOWN`? Have you checked if games doesn't read these locations in any way or form, even in test mode?

Reply:
Didn't really know that ipt unused and unknown had a slight but major difference so going to change these accordingly. Other than that, the seattle_analog games all consist of racing games which I noticed IN2 (normally controls players 3 and 4 for the joystick games) are left unconnected for these games.